_health-care/_js/components/Nav.jsx

```diff
@@ -12,116 +11,88 @@ class Nav extends React.Component {
+              </li>
+              <li className={this.props.currentUrl === '/military-service/additional-information' ? ' section-current' : ''}>
+                  Additional Military Information
+              </li>
+            </ul>
+          </div>
         </li>
-        <li role="presentation">
-          <Link to="/financial-assessment" className={this.props.currentUrl.startsWith('/financial-assessment/') ? ' usa-current' : ''}>
-            Financial Assessment
-          </Link>
-          <ul className="usa-sidenav-sub_list">
-            <li>
-              <Link to="/financial-assessment/financial-disclosure" activeClassName="usa-current">
+        <li role="presentation" className="step five wow fadeIn animated">
+          <div>
+            <h5 className={this.props.currentUrl.startsWith('/financial-assessment') ? ' section-current' : ''}>Financial Assessment</h5>
+            <ul className="usa-unstyled-list">
+              <li className={this.props.currentUrl === '/financial-assessment/financial-disclosure' ? ' section-current' : ''}>
                 Financial Disclosure
-              </Link>
-            </li>
-            <li>
-              <Link to="/financial-assessment/spouse-information" activeClassName="usa-current">
-                Spouse Information
-              </Link>
-            </li>
-            <li>
-              <Link to="/financial-assessment/child-information" activeClassName="usa-current">
-                Child Information
-              </Link>
-            </li>
-            <li>
-              <Link to="/financial-assessment/annual-income" activeClassName="usa-current">
-                Annual Income
-              </Link>
-            </li>
-            <li>
-              <Link to="/financial-assessment/deductible-expenses" activeClassName="usa-current">
-                Deductible Expenses
-              </Link>
-            </li>
-          </ul>
+              </li>
+              <li className={this.props.currentUrl === '/financial-assessment/spouse-information' ? ' section-current' : ''}>
+                  Spouse Information
+              </li>
+              <li className={this.props.currentUrl === '/financial-assessment/child-information' ? ' section-current' : ''}>
+                  Child Information
+              </li>
+              <li className={this.props.currentUrl === '/financial-assessment/annual-income' ? ' section-current' : ''}>
+                  Annual Income
+              </li>
+              <li className={this.props.currentUrl === '/financial-assessment/deductible-expenses' ? ' section-current' : ''}>
+                  Deductible Expenses
+              </li>
+            </ul>
+          </div>
         </li>
-        <li role="presentation">
-          <Link to="/review-and-submit" activeClassName="usa-current">
-            Review and Submit
-          </Link>
+        <li role="presentation" className="step six wow fadeIn animated">
```

Comment:
Just noticed we might want to add the `last` class to this so we don't have another line extending down from 6, but we can do that in the next PR when we improve the steps anyway!